pkg/backup/backup_test.go

```diff
@@ -411,7 +411,7 @@ func TestBackup(t *testing.T) {
 				certificatesGroup: nil,
 				rbacGroup:         errors.New("rbac error"),
 			},
-			expectedError: errors.New("[v1 error, rbac error]"),
+			expectedError: nil,
```

Comment:
This test doesn't have any case that represents a backup failure. It would be nice if we could use a few cases to document what those scenarios would be,